templates/libraries/_tool_vertical_list_item.html

```
@@ -0,0 +1,22 @@
+{% if tool.url %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)